PyRDF/backend/Dist.py

```diff
@@ -278,6 +278,9 @@ def execute(self, generator):
         treename = generator.head_node.get_treename()
         selected_branches = generator.head_node.get_branches()
 
+        # Avoid having references to self inside the mapper
+        initialization = self.initialization
```

Comment:
This will raise an `AttributeError` if `self` has not `initialization` attribute. Can we perhaps, in the constructor of every backend, assign `initialization` to an empty function, so that it is a no-op if the user has not specified any initialization function. This way we also avoid the `try` `catches` below.

Reply:
>  Can we perhaps, in the constructor of every backend, assign initialization to an empty function

I really like this approach. I will put `self.initialization = lambda: None` on the `Backend` metaclass so we solve the issue and remove the `try-catch`.